### ISSUE TITLE
Create example app

### ISSUE THREAD

**Issue by pors** [reactionic/reactionic#19]
Next to the demo[ kitchen sink](https://github.com/pors/reactionic-kitchensink) app it would be cool to have one or more example apps. Some ideas:
- [Todo app](https://www.meteor.com/tutorials/react/creating-an-appl)
- [Contacts](https://github.com/meteoric/contacts)
- [Product hunt](https://github.com/meteoric/meteorhunt)
- Other ideas?

If anyone would like to give it a shot, that'd be awesome!

**Comment by gabrielseco** [comment]
I think on monday, I would have some to time to do something.
I'll fork the kitchen sink app and I'll give it a try

**Comment by johnslemmer** [comment]
@pors at the same time that I have been doing some React 15 upgrading I have also been working on this https://github.com/johnslemmer/todos-reactionic-meteor example app that is trying to replicate https://github.com/meteor/module-todos-react example app.  It is far from ready but it is a start.  I'll be plugging away on it over the next little bit.  Let me know how I can help.

**Comment by pors** [comment]
@johnslemmer awesome John! I don't know how much time you have to work on this, but what would be really cool is to have a blog post explaining how you did it, like what Sam Corcos did here: https://medium.com/@SamCorcos/meteor-react-ionic-mobile-app-part-1-the-basic-template-9355ebf3397f#.a5fy89z20. 

No need for it of course, just an idea :)

Lemme know if I can help with anything.

**Comment by johnslemmer** [comment]
Not in the near future at least. Maybe later in May I'll have the cycles to really do good things for this example.

**Comment by johnslemmer** [comment]
OK.  It isn't a full app example like I wanted to do.  But I made a pretty thorough [boilerplate](https://github.com/johnslemmer/boilerplate-reactionic-meteor) that I hope helps people get started.  Should especially be good for people pretty unfamiliar with this whole technology stack.

**Comment by gabrielseco** [comment]
Hi everyone.

I have my app on my repo, it's pretty simple indeed.

It uses the http://www.brewerydb.com/ but in freemium mode, it limits too much.

I used meteor server to do the API calls to handle CORS.

I would need someone to do the code revision.

I think after that, if you guys want can we upload to the organization's repo.

If anyone wants to do it, I leave it here.

https://github.com/gabrielseco/brewery-app

**Comment by pors** [comment]
Wow, guys, this is awesome! Both the boilerplate and the brewery app should be mentioned in the README, the GUIDE and the main project page IMO. Would you like me to do that?

@gabrielseco you can move the repo into the organization yourself (I can't do that): https://help.github.com/articles/transferring-a-repository/#transferring-from-a-user-to-an-organization
It would also be nice to have it hosted somewhere. Unfortunately, MDG doesn't host if for free anymore, but maybe Heroku is an alternative? https://crater.io/posts/Ytf4qeKLxG8dschsj/no-more-free-meteor-hosting
If you have your own server that might be a solution as well, I have done that with the kitchen-sink.

**Comment by johnslemmer** [comment]
Yeah, link away.  Also, we can transfer the boilerplate to the organization
as well if you would like

**Comment by gabrielseco** [comment]
I'll try heroku this weekend then and when I have it, I'll transfer

**Comment by pors** [comment]
@gabrielseco cool! I'll check it out in more detail after my holidays.

@johnslemmer yeah it would be great if we can put it in the organization. You are the only one who can do that, so please do. I'll link when it has found its new home :)

**Comment by johnslemmer** [comment]
tried to transfer.  Got this error.

> You don’t have admin rights to reactionic

**Comment by pors** [comment]
Ah I only added you as a collaborator of the reactionic repo, not the entire organization. You should now have received an invitation

**Comment by johnslemmer** [comment]
OK.  Sorry was out of touch the last few days.  It is now transferred over to the org.

**Comment by DarrylD** [comment]
I plan to convert one of my ionic apps I could add when finished.

**Comment by pors** [comment]
@DarrylD awesome! Let us know if we can help

**Comment by DarrylD** [comment]
made a quick demo and added redux https://github.com/DarrylD/react-ionic-redux
